lib/uploadcare/resources/file.rb

```diff
@@ -95,6 +95,7 @@ def is_deleted?
       # copy file to target location
       # note what file copied with operations
       def copy with_operations=true, target=nil
+        warn "[DEPRECATION] `copy` is deprecated.  Please use `internal_copy` or `external_copy` instead."
```

Comment:
deprecated and will be removed in version 3.0

what about http://ruby-doc.org/stdlib-2.2.2/libdoc/rubygems/rdoc/Gem/Deprecate.html?